packages/@aws-cdk/aws-ivs/lib/channel.ts

```diff
@@ -0,0 +1,145 @@
+import * as core from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnChannel } from './ivs.generated';
+
+/**
+ * Represents an IVS Channel
+ */
+export interface IChannel extends core.IResource {
+  /**
+   * The channel ARN. For example: arn:aws:ivs:us-west-2:123456789012:channel/abcdABCDefgh
+   *
+   * @attribute
+   */
+  readonly channelArn: string;
+}
+
+/**
+ * Reference to a new or existing IVS Channel
+ */
+abstract class ChannelBase extends core.Resource implements IChannel {
+  // these stay abstract at this level
+  public abstract readonly channelArn: string;
+}
+
+/**
+  Channel latency mode
+*/
+export enum LatencyMode {
+  /**
+   * Use LOW to minimize broadcaster-to-viewer latency for interactive broadcasts.
+   */
+  LOW = 'LOW',
+
+  /**
+   * Use NORMAL for broadcasts that do not require viewer interaction.
+   */
+  NORMAL = 'NORMAL',
+}
+
+/**
+  * The channel type, which determines the allowable resolution and bitrate.
+  * If you exceed the allowable resolution or bitrate, the stream probably will disconnect immediately.
+*/
+export enum ChannelType {
+  /**
+   * Multiple qualities are generated from the original input, to automatically give viewers the best experience for
+   * their devices and network conditions.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ivs-channel.html
+   */
+  STANDARD = 'STANDARD',
+
+  /**
+   * delivers the original input to viewers. The viewer’s video-quality choice is limited to the original input.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ivs-channel.html
+   */
+  BASIC = 'BASIC',
+}
+
+/**
+ * Properties for creating a new Channel
+ */
+export interface ChannelProps {
+  /**
+   * Whether the channel is authorized. Default: false
+   *
+   * @default false
+   */
+  readonly authorized?: boolean;
+
+  /**
+   * Whether the channel is authorized. Default: false
+   *
+   * @default LatencyMode.LOW
+   */
+  readonly latencyMode?: LatencyMode;
+
+  /**
+   * Channel name.
+   *
+   * @default Empty String
```

Comment:
```suggestion
   * @default - None
```

does CloudFormation generate a name if one is not provided?

Reply:
No